### PR TITLE
chore(deps): bump expo sdk from 41 to 42 (#153)

### PR DESCRIPTION
**Pull request by tiagoheliob**
* Upgrading expo packages
* Fixing toast implementation after update

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,12 +12,14 @@ import Main from './Main'
 export default function App() {
   return (
     <AuthenticationProvider>
-      <SecretsProvider>
-        <PaperProvider theme={theme}>
-          <Main />
-          <StatusBar style="auto" />
-        </PaperProvider>
-      </SecretsProvider>
+      <RootSiblingParent>
```
Comment (simoneb) [inline src/App.js:15]: can you briefly explain this change or point to the docs which explain why it's necessary?

**Review by simoneb — APPROVED** [review]
LGTM, see 1 inline comment

File: src/App.js (hunk 1 above)
Comment (tiagoheliob) [reply src/App.js:15]: Yes, when updating the expo it also has updated the clipboard package,  you can check this expo section https://docs.expo.dev/ui-programming/react-native-toast/#cross-platform--react-native-root-toast

Comment (simoneb) [reply src/App.js:15]: Makes sense, let's leave this comment here for reference. Looks good, thanks.

**Pull request merged**